edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/Agent.cs

```diff
@@ -148,30 +152,30 @@ public async Task ReconcileAsync(CancellationToken token)
                     }
                     else
                     {
-                        status = Option.Some(DeploymentStatus.Success);
+                        status = DeploymentStatus.Success;
```

Comment:
Should we invert the logic here, so this is closer to the reason why it's Success?  Plus I think it looks better:
```
if (deploymentConfig == DeploymentConfig.Empty)
{
    status = DeploymentStatus.Success;
}
else
{
// [...]
    if (plan.IsEmpty)
    {
        status = DeploymentStatus.Success;
    }
```

Reply:
will do it in master.